ci-operator/step-registry/stackrox/qa-e2e/stackrox-qa-e2e-commands.sh

```
@@ -28,8 +28,8 @@ function mapTestsForComponentReadiness() {
         results_file="${1}"
         echo "Patching Tests Result File: ${results_file}"
         if [ -f "${results_file}" ]; then
-            echo "Mapping Test Suite Name To: ACSLatest-lp-interop"
-            /tmp/bin/yq eval --output-format xml -iI0 '.testsuite."+@name" = "ACSLatest-lp-interop"' "$results_file" || echo "Warning: yq failed for ${results_file}, debug manually" >&2
+            echo "Mapping Test Suite Name To: ${REPORTPORTAL_CMP}"
+            /tmp/bin/yq eval --output-format xml -iI0 '.testsuite.+@name=env(REPORTPORTAL_CMP)' $results_file || echo "Warning: yq failed for ${results_file}, debug manually" >&2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)